src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {
+  Link,
+  useLocation,
+  useParams,
+  useResolvedPath,
+} from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  allPeople: Person[];
+};
+
+export const PersonInfo: React.FC<Props> = ({ person, allPeople }) => {
+  const {
+    slug,
+    name,
+    died,
+    born,
+    sex,
+    motherName,
+    fatherName,
+  } = person;
+  const { personId } = useParams();
+  const location = useLocation();
+  const parentPath = useResolvedPath('../').pathname;
+
+  const isSelected = person.slug === personId;
+  const findFather = allPeople.find(human => (
```

Comment:
Maybe it is better to rename the variable to "father" or "foundFather" because "find" is just a verb. The same about "findMother"